src/modules/roc_fec/reader.cpp

```diff
@@ -374,7 +322,7 @@ void Reader::update_source_block_() {
 
         if (packet::blknum_lt(fec.source_block_number, cur_sbn_)) {
             roc_log(LogTrace,
-                    "fec reader: dropping source packet: belongs to previous block:"
+                    "fec reader: dropping source packet from  previous block:"
```

Comment:
Extra space between `from` and `previous`.

Reply:
Fixed.